src/common/logger/ShortlistingLogger.ts

```
@@ -0,0 +1,333 @@
+  }
+
+  /**
+   * Logs a shortlisting failure to both CSV file and main application logs
+   * @param failure - The failure details to log
+   */
+  static logFailure(failure: ShortlistingFailureLog) {
+    try {
+      // Ensure logging system is initialized
+      this.initialize();
+
+      const logFilePath = path.join(this.LOG_DIR, this.FAILURE_LOG_FILE);
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)